src/igvfd/schemas/auxiliary_set.json

```diff
@@ -78,6 +82,10 @@
                 "title": "Construct Library",
                 "type": "string",
                 "linkTo": "ConstructLibrary"
+            },
```

Comment:
You can remove if you like, import_items property

Reply:
done